node_modules/@okta/courage/package.json

```diff
@@ -20,7 +20,7 @@
     "type": "version"
   },
   "_requiredBy": [
-    "#USER"
```

Comment:
actually wasn't quite sure why I got this but travis failed me because of this extra config.
I upgrade courage via install-shrinkwrap tool.